<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thomas Vogt</author>
 <class>Arinc665Qt::Media::LoadWidget</class>
 <widget class="QWidget" name="Arinc665Qt::Media::LoadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbLoad">
     <property name="title">
      <string>Load</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="partNumberLabel">
        <property name="text">
         <string>Part Number</string>
        </property>
        <property name="buddy">
         <cstring>partNumber</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="partNumber"/>
      </item>
      <item row="2" column="1">
       <widget class="QTreeView" name="targetHardwareIdsPositions">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTargetHardwaresPositions">
        <property name="text">
         <string>Target Hardwares / Postitions</string>
        </property>
        <property name="buddy">
         <cstring>targetHardwareIdsPositions</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTableView" name="dataFiles">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTableView" name="supportFiles">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblDataFiles">
        <property name="text">
         <string>Data Files</string>
        </property>
        <property name="buddy">
         <cstring>dataFiles</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblSupportFiles">
        <property name="text">
         <string>Support Files</string>
        </property>
        <property name="buddy">
         <cstring>supportFiles</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
